lib/rake/circle.rake

```diff
@@ -140,13 +140,18 @@ namespace :circle do
     end
   end
 
+  desc 'Rebuild dashboard assets with updated locals.yml settings before running UI tests'
   task :recompile_assets do
     if CircleUtils.tagged?(SKIP_UI_TESTS_TAG)
       ChatClient.log "Commit message: '#{CircleUtils.circle_commit_message}' contains [#{SKIP_UI_TESTS_TAG}], skipping UI tests for this run."
       next
     end
 
+    # TODO: (Brad) Remove when we're satisfied that pegasus host is set correctly.
+    puts "override_pegasus: #{CDO.override_pegasus}"
+
     Dir.chdir(dashboard_dir) do
+      RakeUtils.rake 'tmp:cache:clear'
```

Comment:
@islemaster note that we could instead run `rm -rf tmp/cache/assets` instead of this Rake task to remove only the Sprockets cache dir rather than the whole `tmp/cache`, which might fix the Circle warning about the missing `tmp/cache` directory.

Reply:
That seems like a good change.

Experimenting with this a little more on my local machine, it seems like we might be able to save a few more seconds with a more targeted recompile:

```
rm -rf tmp/cache/assets
rake assets:precompile_application_js
```

I'll propose this in a separate PR.